projects/baby_driver/scripts/can_message.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# coding: utf-8
+"""python can message with 8 bytes"""
+from __future__ import print_function
+
+import can
+
+def send_message(msg):
+    """sends can message with 8 bytes. You can change the 2nd one"""
+    #change the channel later to can0 or vcan0
```

Comment:
stylistic nit: please put a space after the #, though I'm not sure the comment is necessary. You should probably make the channel a keyword argument to `send_message` (like `channel="can0"`).

also some docstring formatting suggestions that were used in Arshan's can_dump project are here: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings

Reply:
We could but right now the make babydriver is for the stm32. I could try to setup it with x86 though, I'll need to figure out a way to prompt another command window.

Cool, I'll check it out. Thanks!